Decred Wallet/pt-BR.lproj/Localizable.strings

```diff
@@ -71,10 +76,13 @@
 "pinsDidNotMatch" = "PINs não coincidem. Tente novamente.";
 "pin" = "PIN";
 "pinStrength" = "FORÇA DO PIN";
+"promptStartupPIN" = "Enter Current Startup PIN";
+"next" = "Next";
+"create" = "Create";
+"back" = "Back";
 
 /* Change Spending PIN/Password */
 "enterCurrentSpendingPassword" = "Insira Senha De Gasto Atual";
-"changingSpendingPINPass" = "Changing spending %@...";
```

Comment:
Should be replaced with `"enterCurrentSpendingPIN" = "Enter Current Spending PIN";`